DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: modulartabler
 Type: Package
 Title: Building count tables from parts
-Version: 0.1.0
+Version: 0.1.1
```

Comment:
Sure thing, given it is a new package and we aren't publishing any release builds (yet), happy to continue stacking on 0.1! I think there's a few things that need solving before it will be accepted on CRAN, will need to read the [docs](https://cran.r-project.org/submit.html) again. Would also be good I think to get a vignette in the package (or alternatively a github pages built from [pkgdown](https://github.com/r-lib/pkgdown)).

Reply:
RE a vignette - can you think of any suitable, publicly-available datasets that would be good to test this package out on?

RE CRAN submission - I'm getting my head around them now. Planning on focusing on getting maskr up first because it's a simpler package than this one, and will be useful for this one

Reply:
> RE a vignette - can you think of any suitable, publicly-available datasets that would be good to test this package out on?

Good question, I think a suitable dataset would ideally need to meet the following criteria:

* Simple and self-evident variables
* Combination of categorical and continuous data for Base/Range mapping tables
* Small (and large) enough that cell suppression (primary & secondary) can be demonstrated easily
* Optional: present in either a base package or a common package

There's a few that might meet some of this criteria - `survival` has some good ones but are really best suited for survival analysis, not aggregation. I'l have a think 🤔

Reply:
Reviewed `data()` for the in-built R datasets and wasn't wowed. May need to source elsewhere

Reply:
There's a good list of datasets [here](https://vincentarelbundock.github.io/Rdatasets/articles/data.html).

Sorting by highest number of rows (because these are most likely to be 1 person/row), the [`military`](https://vincentarelbundock.github.io/Rdatasets/doc/openintro/military.html) dataset looks quite simple, and [`flights`](https://vincentarelbundock.github.io/Rdatasets/doc/nycflights13/flights.html) is another option.

If we wanted something health-related, there's [`simpsons_paradox_covid`](https://vincentarelbundock.github.io/Rdatasets/doc/openintro/simpsons_paradox_covid.html), but this only has 3 variables. Alternatively, [`Fertility`](https://vincentarelbundock.github.io/Rdatasets/doc/AER/Fertility.html) is a bit more complicated and includes race information.

At any rate, I can close this comment, and will open a new issue for this.